src/io/flutter/inspector/InspectorService.java

```diff
@@ -229,6 +230,56 @@ public void addClient(InspectorServiceClient client) {
     return getListHelper(instanceRef, "getProperties");
   }
 
+  /**
+   * If the widget tree is not ready, the application should wait for the next
+   * Flutter.Frame event before attempting to display the widget tree. If the
+   * application is ready, the next Flutter.Frame event may never come as no
+   * new frames will be triggered to draw unless something changes in the UI.
+   */
+  public CompletableFuture<Boolean> isWidgetTreeReady() {
+    return hasServiceMethod("isWidgetTreeReady").<Boolean>thenComposeAsync((Boolean hasMethod) -> {
```

Comment:
Can you add a TODO: to delete this code at some point? I think we can safely remove after one or two revs of the Flutter `alpha` channel.

Reply:
(the `hasServiceMethod("isWidgetTreeReady")` code)

Reply:
done.